src/common.c

```diff
@@ -66,6 +66,73 @@ ly_strncmp(const char *refstr, const char *str, size_t str_len)
     }
 }
 
+uint64_t
```

Comment:
I mean, it probably does not matter but this is redundant, `uint32_t` is more than enough.